src/components/button/button.js

```diff
@@ -2,7 +2,13 @@ import styled from 'styled-components';
 
 export default function Button({disabled, buildId, buildTime, onHandleClick, buttonText}) {
   return (
-    <StyledButton type='button' disabled={disabled} data-buildid={buildId} data-buildtime={buildTime} onClick={onHandleClick}>
+    <StyledButton
+      type="button"
+      disabled={disabled}
+      data-buildid={buildId}
+      data-buildtime={buildTime}
```

Comment:
What is the purpose of the `data-...` attributes? It should not be necessary to store data in DOM elements like this.

Reply:
I understand this now
you mean to say that I should remove the data-... and then insert it directly into the onActionButtonClick ?

I pass only the parameters. but yes now that I write this. sounds logical

Reply:
thanks, done